src/arch/arm/isa/insts/sve.isa

```diff
@@ -4249,19 +4249,19 @@ let {{
     sveTerInst('mls', 'Mls', 'SimdMultAccOp', signedTypes, mlsCode)
     # ADCLT
     adcltCode = 'res = srcElem1 + srcElem2 + carryIn;'
-    sveTerInstUnpred('adclt', 'Adclt', 'VectorIntegerArithOp', unsignedTypes,
+    sveTerInstUnpred('adclt', 'Adclt', 'SimdAddOp', unsignedTypes,
```

Comment:
I think SVE is considered "vector" since it's variable length. But I'll let @giactra chime in here with his opinion.

Reply:
@giactra any opinions here?

Reply:
@giactra are you going to be able to look at this before the staging branch is created on Friday?